src/main/frontend/src/components/landingpage/LandingPageItem.js

```diff
@@ -0,0 +1,44 @@
+import classes from "./LandingPageItem.module.css";
+import ItemImage from "../../assets/lowersectpic.png";
+
+function LandingPageItem(props) {
+  if (props.item_type === "SMALL") {
+    return (
+      <div className={classes.small_item_box}>
+        <div className={classes.small_image_container}>
+          <img className={classes.full_size_picture} src={ItemImage} />
+        </div>
+        <h3 className={classes.cardtitle}>{props.card_title}</h3>
+        <p className={classes.carddescription}>
+          Start from <span className={classes.cardprice}>${props.price}</span>
+        </p>
+      </div>
```

Comment:
```suggestion
      <div className={classes.[`${props.item_type.toLowecase()}_item_box`]}>
        <div className={classes[[`${props.item_type.toLowecase()}_image_container`]}>
          <img className={classes.full_size_picture} src={ItemImage} />
        </div>
        <h3 className={classes.cardtitle}>{props.card_title}</h3>
        <p className={classes.carddescription}>
          Start from <span className={classes.cardprice}>${props.price}</span>
        </p>
      </div>
```
this way there is no need for duplicating code, and if else... cases can be removed